### ISSUE TITLE
pycparser 2.21 breaks generation

### ISSUE THREAD

**Issue by Munken** [vberlier/narmock#25]
The example/basic breaks with the latest pycparser version:
```
  File "/home/munken/.local/bin//narmock", line 8, in <module>
    sys.exit(main())
  File "/home/munken/.local/lib/python3.8/site-packages/narmock/cli.py", line 44, in main
    generate_mocks(expanded_code=args.g.read(), directory=args.d, keep_args=args.k)
  File "/home/munken/.local/lib/python3.8/site-packages/narmock/api.py", line 16, in generate_mocks
    generator.add_mock(function)
  File "/home/munken/.local/lib/python3.8/site-packages/narmock/generator.py", line 184, in add_mock
    self.mocks.append(GeneratedMock(mocked_function))
  File "/home/munken/.local/lib/python3.8/site-packages/narmock/generator.py", line 113, in __init__
    self.return_value_decl = decl(
  File "/home/munken/.local/lib/python3.8/site-packages/narmock/generator.py", line 19, in decl
    return node.Decl(name, [], [], [], type, None, None)
TypeError: __init__() missing 1 required positional argument: 'bitsize'

```
This is caused by pycparser adding another argument to Decl:
https://github.com/eliben/pycparser/pull/435

Steps to reproduce
```
 % git clone https://github.com/vberlier/narmock.git              /tmp/bla P14s
Cloning into 'narmock'...
remote: Enumerating objects: 835, done.
remote: Counting objects: 100% (69/69), done.
remote: Compressing objects: 100% (69/69), done.
remote: Total 835 (delta 35), reused 0 (delta 0), pack-reused 766
Receiving objects: 100% (835/835), 188.08 KiB | 6.27 MiB/s, done.
Resolving deltas: 100% (579/579), done.
 % cd narmock/examples                                            /tmp/bla P14s
 % cd basic                                      narmock/examples (master) P14s
 % python3 -m venv venv                          examples/basic (master) P14s
 % source venv/bin/activate
 % pip install narmock                             examples/basic (master) P14s
Requirement already satisfied: narmock in /home/munken/.local/lib/python3.8/site-packages (0.2.12)
Requirement already satisfied: pycparser<3.0,>=2.19 in /home/munken/.local/lib/python3.8/site-packages (from narmock) (2.21)
Requirement already satisfied: jinja2<3.0,>=2.10 in /usr/lib/python3/dist-packages (from narmock) (2.10.1)
 % touch main.c                                    examples/basic (master) P14s
 % make                                            examples/basic (master) P14s
cc -E main.c | narmock -g
Traceback (most recent call last):
  File "/home/munken/.local/bin//narmock", line 8, in <module>
    sys.exit(main())
  File "/home/munken/.local/lib/python3.8/site-packages/narmock/cli.py", line 44, in main
    generate_mocks(expanded_code=args.g.read(), directory=args.d, keep_args=args.k)
  File "/home/munken/.local/lib/python3.8/site-packages/narmock/api.py", line 16, in generate_mocks
    generator.add_mock(function)
  File "/home/munken/.local/lib/python3.8/site-packages/narmock/generator.py", line 184, in add_mock
    self.mocks.append(GeneratedMock(mocked_function))
  File "/home/munken/.local/lib/python3.8/site-packages/narmock/generator.py", line 113, in __init__
    self.return_value_decl = decl(
  File "/home/munken/.local/lib/python3.8/site-packages/narmock/generator.py", line 19, in decl
    return node.Decl(name, [], [], [], type, None, None)
TypeError: __init__() missing 1 required positional argument: 'bitsize'
make: *** [Makefile:16: __mocks__.h] Error 1

```

**Comment by Munken** [comment]
Forcing 2.19 it works
```
python3 -m venv venv
pip install pycparser==2.19
pip install narmock
```

**Comment by vberlier** [comment]
Thanks for noticing! Should be fixed in `v0.2.13` :)
